### PR TITLE
[Keyboard] Adding Styrka Topre PCB

### PR DESCRIPTION
**Pull request by blindassassin111**
## Description

Adding support for the Styrka Topre PCB.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/viktus/styrka_topre/keymaps/default/readme.md
Comment (waffle87) [inline keyboards/viktus/styrka_topre/keymaps/default/readme.md:1]: This file should be removed.

**Review by waffle87 — APPROVED** [review]

File: keyboards/viktus/styrka_topre/keymaps/default/readme.md
Comment (blindassassin111) [reply keyboards/viktus/styrka_topre/keymaps/default/readme.md:1]: Oops, knew I forgot something in the update 😅

**Comment by blindassassin111** [comment]
Any changes still needed?

**Comment by blindassassin111** [comment]
Bumping this again as it has been a while.

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**